package.json

```diff
@@ -21,9 +21,11 @@
     "@angular/platform-browser": "4.1.3",
     "@angular/platform-browser-dynamic": "4.1.3",
     "@ionic-native/core": "3.12.1",
+    "@ionic-native/geolocation": "^4.2.1",
```

Comment:
We dont need this too 😄   `npm uninstall --save @ionic-native/geolocation`

Reply:
GREAT! done!